sky/packages/updater/lib/main.dart

```diff
@@ -3,27 +3,45 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
+import 'dart:math';
 import 'dart:io';
+import 'dart:typed_data';
 
 import 'package:mojo/core.dart';
 import 'package:flutter/services.dart';
-import 'package:sky_services/updater/update_service.mojom.dart';
+// TODO(mpcomplete): the following breaks unless I import 'as updater'. Why?
+import 'package:sky_services/updater/update_service.mojom.dart' as updater;
```

Comment:
Please don't check in a comment with an unanswered question... either figure out the problem, or file a bug on the dart sdk and replace the comment with something like "This is imported using 'as updater' because otherwise the compiler says ...." and then point to the corresponding dart bug.